configs/webpack.dev.js

```diff
@@ -3,11 +3,12 @@ const baseConfig = require("@mendix/pluggable-widgets-tools/configs/webpack.conf
 
 const customConfig = {
     // Custom configuration goes here
-    devtool: "source-map"
+    devtool: "source-map",
+    externals: ["mendix"]
 };
 const previewConfig = {
     // Custom configuration goes here
-    devtool: "source-map"
+    devtool: "inline-source-map"
```

Comment:
Why do you use inline here?

Reply:
for Design mode. It doesnt support source-map